src/main/java/org/ibp/api/rest/germplasm/EntryTypeResource.java

```diff
@@ -0,0 +1,38 @@
+package org.ibp.api.rest.germplasm;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.generationcp.middleware.domain.dms.Enumeration;
+import org.ibp.api.java.entrytype.EntryTypeService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+@Api(value = "Entry Type Services")
+@Controller
+@RequestMapping("/crops")
+public class EntryTypeResource {
+
+	@Resource
+	private EntryTypeService entryTypeService;
+
+	@ApiOperation(value = "Get Study Entry Types",
```

Comment:
Let's just simply call as entry types: Use "Get Entry types" and the method name as `getEntryTypes`.